cmd/airgap/listimages.go

```diff
@@ -39,7 +40,7 @@ func NewAirgapListImagesCmd() *cobra.Command {
 			}
 
 			if opts.EnableDynamicConfig {
-				return fmt.Errorf("dynamic config is not supported for airgap list-images")
+				return errors.New("dynamic config is not supported for airgap list-images")
```

Comment:
(The `err113` linter would complain about these dynamic errors with a static message)